### PR TITLE
RIA-3390:Request Home Office data flow and negative scenario

### PR DESCRIPTION
**Pull request by divisathyan**
RIA-3390:Request Home Office data flow and negative scenario

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3390

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by jkumar-moj — APPROVED** [review]
looks good. 👍

**Comment by jkumar-moj** [comment]
@divisathyan Could you take out the RIA-3569 changes to break the dependency considering it is still in progress and will block this ticket from putting it for ready for testing. Unless there is any reason to be dependent in which case we should wait for that ticket as well.

**Comment by divisathyan** [comment]
Authorization Header was added and I used the open ID code. We can plan how we will move RIA-3569

**Review by jkumar-moj — APPROVED** [review]

File: src/functionalTest/resources/scenarios/RIA-3390-request-home-office-data-success.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "errors": [],
+    "caseData": {
+      "template": "minimal-appeal-submitted.json",
+      "replacements": {
+        "homeOfficeReferenceNumber": "1212-0099-0062-8083",
+        "homeOfficeSearchStatus": "SUCCESS"
+      }
+    }
+  }
+}
```
Comment (Srihari-Nakkala) [inline src/functionalTest/resources/scenarios/RIA-3390-request-home-office-data-success.json:28]: @divisathyan we need to cover this scenario as well **error message when event has already retrieved the data* *successfully***

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Pull request merged**